src/Symfony/Component/VarDumper/Caster/StubCaster.php

```diff
@@ -89,4 +89,12 @@ public static function castScalar(ScalarStub $scalarStub, array $a, Stub $stub):
 
         return $a;
     }
+
+    public static function castVirtualHookedProperty(VirtualHookedPropertyStub $hookedStub, array $a, Stub $stub): array
+    {
+        $stub->type = Stub::TYPE_SCALAR;
+        $stub->attr['value'] = 'virtual hooked property'.($hookedStub->propertyType ? ' of type "'.$hookedStub->propertyType.'"' : '');
```

Comment:
```suggestion
        $stub->attr['value'] = 'virtual property'.($hookedStub->propertyType ? ' of type "'.$hookedStub->propertyType.'"' : '');
```

The "hooked" word looks redundant to me.

Reply:
Suits me fine